libraries/AC_Fence/AC_PolyFence_loader.cpp

```diff
@@ -6,7 +6,7 @@
 #include <AP_BoardConfig/AP_BoardConfig.h>
 
 #ifndef AC_FENCE_DUMMY_METHODS_ENABLED
-#define AC_FENCE_DUMMY_METHODS_ENABLED  (!(APM_BUILD_TYPE(APM_BUILD_Rover) | APM_BUILD_COPTER_OR_HELI | APM_BUILD_TYPE(APM_BUILD_ArduPlane) | APM_BUILD_TYPE(APM_BUILD_ArduSub) | (AP_FENCE_ENABLED == 1)))
```

Comment:
Please add a comment so people update the other place we define this when they change this line